include/zephyr/dt-bindings/regulator/da1469x.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Jerzy Kasenberg
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_DA1469X_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_DA1469X_H_
+
+/*
+ * Set this mode if rail should be powered during sleep,
+ * mode applies to VDD, V18, V18P
+ */
+#define DA1469X_MODE_SLEEP_LDO					0x01
+/* Settings valid for VDD, V14, V18, V18P */
+#define DA1469X_MODE_DCDC_HIGH_BATTERY_VOLTAGE_ON		0x02
+#define DA1469X_MODE_DCDC_LOW_BATTERY_VOLTAGE_ON		0x04
+/* Settings valid for V30 */
+#define DA1469X_MODE_LDO_VBUS_ON				0x08
+#define DA1469X_MODE_LDO_VBAT_ON				0x10
+#define DA1469X_MODE_V30_VBAT_CLAMP_ON				0x20
```

Comment:
see my previous comment, do not look as valid modes

Reply:
DA1469X_MODE_DCDC_HIGH_BATTERY_VOLTAGE_ON means that rail should used DCDC when VBAT level is consider high, DA1469X_MODE_DCDC_LOW_BATTERY_VOLTAGE_ON means that DCDC can be used below this level. If both bits are 0 rail will not use DCDC.
DA1469X_MODE_SLEEP_LDO means that rail is powered during sleep.